src/mcts/search.cc

```diff
@@ -1540,18 +1544,26 @@ void SearchWorker::DoBackupUpdateSingleNode(
     }
     n->FinalizeScoreUpdate(v / (1.0f + params_.GetShortSightedness() * depth),
                            d, m, node_to_process.multivisit);
+    if (n_to_fix > 0 && !n->IsTerminal()) {
+      n->AdjustForTerminal(
+          v_delta / (1.0f + params_.GetShortSightedness() * depth), d_delta,
```

Comment:
Unless we have plans to do something else with ShortSighteness, it would make sense to revert it.

Reply:
I strongly vote for reverting ShortSightedness, as the problem it originally attempted to solve (depth dependent adjustment of drawish scores) didn't work out the way it was hoped to, and the moves left head so far seems to work good enough.

Reply:
I'm on board with reverting short sightedness - but that will be a separate PR.
I think I need to run a bigger test on this before I'm willing to be happy that it has 0 regression risk to offset the training data quality gains.

Reply:
Yeah I didn't suggest to revert it now, it probably sounded like I did. :)